python/dask_cudf/dask_cudf/tests/test_distributed.py

```diff
@@ -97,3 +97,19 @@ def test_p2p_shuffle():
                 ddf.compute().sort_values("x"),
                 check_index=False,
             )
+
+
+@pytest.mark.skipif(
+    not more_than_n_gpus(3),
+    reason="Machine does not have more than three GPUs",
+)
+def test_unique():
+    with dask_cuda.LocalCUDACluster(n_workers=3) as cluster:
```

Comment:
Needed >2 workers to reproduce the error locally. Not sure what the problem is yet, but the to-pyarrow dispatch is failing to register on one or more workers.

Reply:
So that means the error is not reproducible by disabling P2P shuffle and this test confirms that, right?

Reply:
This PR does something pretty simple, but the background is slightly confusing:

- Using the latest release of dask, this test would fail without the global `"tasks"` config that is set in this PR
- After https://github.com/dask/dask/pull/11040, this test will also pass when the `"p2p"` shuffle is used
- Even though `"p2p"` works for `dask:main`, I still think it makes sense to use the `"tasks"` default (at least for now). Although `"p2p"` should theoretically be more stable, I have not found this to be the case in practice for GPUs. Also, `"tasks"` is **definitely** faster.